rca/events/models.py

```diff
@@ -11,6 +11,16 @@
 from .blocks import CallToAction, EventDetailPageBlock
 
 
+class EventType(models.Model):
+    title = models.CharField(max_length=100)
+
+    class Meta:
+        ordering = ["title"]
+
+    def __str__(self):
+        return self.title
+
+
```

Comment:
```suggestion
class EventType(models.Model):
    title = models.CharField(max_length=100)
    slug = models.SlugField(blank=True)

    def save(self, *args, **kwargs):
        self.slug = slugify(self.title)
        super(EventType, self).save(*args, **kwargs)
        
    class Meta:
        ordering = ["title"]

    def __str__(self):
        return self.title
```

**Beg**
Could we sluggify this taxonomy as it's going to be used as a filter for the event listing page, and we'll need it.

Reply:
Is there a reason not to slugify in the template for filtering, keep the model simpler?

`<option value={{ event_type|slugify` }}>{{ event_type }}</option>`

Reply:
In the above suggestion, if the user enters a value it gets blatted on save.

Reply:
>Is there a reason not to slugify in the template for filtering, keep the model simpler?

Or if not in the template, in `get_context` when constructing the values to pass to the template. Either option reduces the redundancy of keeping slug on the model.

Reply:
Yea, the reason we slugify these on the model is for when the filtering system, it uses this slug value as a URL arg to filter https://github.com/torchbox/rca-wagtail-2019/pull/686/files#diff-0157f045a04308cf0e5bbc6ce0256711ba1d8bc466a5b54ec6e9c24818e0e39bR274
```
filter_by="subjects__subject__slug__in",
                option_value_field="slug",
```
Feel free to pass on this here, we can pick it up as part of https://projects.torchbox.com/projects/rca-website-rebuild/tickets/787 where we configure the filters

Reply:
Ugh, OK not our usual method of filtering then. Might as well do it here.